results/cmd/api/main.go

```diff
@@ -161,3 +165,34 @@ func (s *server) DeleteTaskRunResult(ctx context.Context, req *pb.DeleteTaskRunR
 	}
 	return nil, nil
 }
+
+// GetTaskRunByID is the helper function to get a TaskRun by results_id
+func GetTaskRunByID(s *server, id string) (*pb.TaskRun, error) {
```

Comment:
```suggestion
func (s *server) getTaskRunByID(s *server, id string) (*pb.TaskRun, error) {
```

Instead of passing in the server as an arg, we can define the func on the struct itself.
I think we can also unexport this func, since this isn't something I would expect external users to call.

Reply:
done